texastribune/donations-app
# (wip) Design round 1

#### What's the plan?

To add styles and functionality per design.
#### What's this PR do?

Adds styles and functionality become a member, benefits, and circle pages.
#### Why are we doing this? How does it help us?

Gets us closer to our design.
#### How should this be manually tested?

Visit the main 'Support Us' page locally, and click around to 'Join Our Circle' and 'Membership Benefits.' Check it out on mobile/tablet too if you like. Use the express checkouts to check they're going to the correct form. Click around on watching the videos, the circle page, the main support us page, the membership benefits page levels grid.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

JS/CSS/HTML is minified and gzipped. Images are compressed. Some images have cache headers set so far, and I'll set more of those as appropriate.
#### What are the relevant tickets?

Sprint task: https://basecamp.com/1866004/projects/1175549/todos/167965443
#### Next steps?

Keep iteratin'
#### Smells?

Don't think so. 
#### TODOs:
- [x] Flesh out mobile
- [x] Add contribution functionality
- [x] Design styles and colors implemented for support, circles, and benefits pages
#### Has the relevant documentation/wiki been updated?

TODO Will add to wiki.
#### Technical debt note

This should lessen our technical debt by moving this outside of our TT app and decreasing the complexity of the JS currently used for join page.
